test/libsolidity/syntaxTests/types/magic_block_basefee_error.sol

```diff
@@ -2,8 +2,14 @@ contract C {
     function f() public view returns (uint) {
         return block.basefee;
     }
+    function g() public view returns (uint ret) {
+        assembly {
+            ret := basefee()
+        }
+    }
 }
 // ====
 // EVMVersion: <=berlin
```

Comment:
```suggestion
// EVMVersion: =berlin
```
Because of https://github.com/ethereum/solidity/pull/11842/files#r695718566, this should be kept ;)

Reply:
True. Just noticed that :) It's fine then.